lib/logflare_web/controllers/api/fallback_controller.ex

```diff
@@ -0,0 +1,18 @@
+defmodule LogflareWeb.Api.FallbackController do
+  use Phoenix.Controller
+  alias Ecto.Changeset
+
+  def call(conn, {:error, %Changeset{} = changeset}) do
+    errors = Changeset.traverse_errors(changeset, fn _, _, {message, _} -> "#{message}" end)
```

Comment:
the `message` will already be a string, no need to type convert.

Reply:
nice call! I had a different message being built 🤦